### PR TITLE
DTSPO-5695 - Diagnostic policy sbox

### PR DESCRIPTION
**Pull request by endakelly**

### CODE REVIEW

File: assignments/mg-dts002/assign.diagnostics.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to set diagnostic setting to forward Activity Logs to an Azure Event Hub",
```
Comment (timja) [inline assignments/mg-dts002/assign.diagnostics.json:6]:
```suggestion
      "description": "Forwards all Activity Logs to an Azure Event Hub",
```

File: assignments/mg-dts002/assign.diagnostics.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      "displayName": "HMCTS Diagnostic Policy",
+      "enforcementMode": "DoNotEnforce",
```
Comment (timja) [inline assignments/mg-dts002/assign.diagnostics.json:8]: why do not enforce?

File: assignments/mg-dts002/assign.diagnostics.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to set diagnostic setting to forward Activity Logs to an Azure Event Hub",
+      "displayName": "HMCTS Diagnostic Policy",
```
Comment (timja) [inline assignments/mg-dts002/assign.diagnostics.json:7]:
```suggestion
      "displayName": "HMCTS Collect Activity Logs",
```

**Review by timja — COMMENTED** [review]
few minor suggestions generally looks good, can you add a README to it though?

File: assignments/mg-dts002/assign.diagnostics.json (hunk 1 above)
Comment (endakelly) [reply assignments/mg-dts002/assign.diagnostics.json]: Updated

File: assignments/mg-dts002/assign.diagnostics.json (hunk 2 above)
Comment (endakelly) [reply assignments/mg-dts002/assign.diagnostics.json:8]: Was keeping it as that just as a placeholder. Will update.

File: assignments/mg-dts002/assign.diagnostics.json (hunk 3 above)
Comment (endakelly) [reply assignments/mg-dts002/assign.diagnostics.json]: Updated

File: assignments/mg-dts002/assign.diagnostics.json (hunk 2 above)
Comment (endakelly) [reply assignments/mg-dts002/assign.diagnostics.json]: Updated

**Review by timja — APPROVED** [review]

**Pull request merged**